README.md

```diff
@@ -58,6 +57,112 @@ Creating machine...
 Starting local Kubernetes cluster...
 ```
 
+## Deploying your first application
+
+You can just deploy our application into your cluster as easily as running following command:
+```
+$  kubectl run hello --image=evalle/gordon:v1.0 --port=8080 --expose
+```
+
+## What just happened?
+
+We asked Kubernetes to deploy our previously build application and we hinted it, that it will be listening on port 8080 and we wanted it to be exposed to a cluster enabling
+load balancing to work.
+
+## How can I access my app?
+
+To access your app we will hook inside Kubernetes network via `kubectl proxy` command, so please open a new terminal and run
+
+```
+$  kubectl proxy
+```
+
+Than you can access your application via following command:
+```
+curl http://localhost:8001/api/v1/namespaces/default/services/hello/proxy/
+```
+
+### Service
+Did you noted a world `service` inside the url? Yes, we are using kubernetes service to access our application. You can see how the service object looks by running:
+```
+$  kubectl describe service hello
+```
+
+Or you can list all other service by:
+```
+$  kubectl get services
+```
+
+Why service is there? The service is very important object in Kubernetes. Its an abstraction for your deployed applications. It helps you to discover your services, to load balance them and
```

Comment:
It helps you to discover your services 
Service helps discover services, it makes sense, but can be confusing, I would use instead
It helps discover your Pods ||  It helps discover your application instances (pods).